mungegithub/mungers/sync/issue-sync.go

```diff
@@ -50,7 +50,7 @@ type Priority int
 
 // String return the priority label in string
 func (p Priority) String() string {
-	return fmt.Sprintf(priorityPrefix+"%d", p)
+	return "priority/failing-test"
```

Comment:
> I did not want to get too deep into removing the auto-prioritization code for this PR unless you requested it. 
I think for now, we can eliminate calls to the auto-pri code paths since we're not doing that anymore.  That said, I think we would like to re-enable (we need to think through a better design) it in the future so don't delete the code entirely (nor should you delete the accompanying Priority code).